core/templates/dev/head/pages/subtopic-viewer-page/navbar-breadcrumb/subtopic-viewer-navbar-breadcrumb.directive.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the navbar breadcrumb of the subtopic viewer.
+ */
+
+require('domain/utilities/UrlInterpolationService.ts');
+require('services/contextual/UrlService.ts');
+
+angular.module('oppia').directive('subtopicViewerNavbarBreadcrumb', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/subtopic-viewer-page/navbar-breadcrumb/' +
+        'subtopic-viewer-navbar-breadcrumb.directive.html'),
+      controller: ['$scope', 'SubtopicViewerBackendApiService', 'UrlService',
+        function($scope, SubtopicViewerBackendApiService, UrlService) {
+          $scope.topicName = UrlService.getTopicNameFromLearnerUrl();
+          $scope.getTopicUrl = function() {
+            return '/topic/' + $scope.topicName;
```

Comment:
Probably should be part of UrlInterpolationService?

Reply:
Done